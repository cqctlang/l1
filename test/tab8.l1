/* default table value */
t = mktab(0);
t[1]++;
printf("%d\n", t[1]);
