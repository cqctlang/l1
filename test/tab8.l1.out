entry: var params, 8 locals, 0 temps
1
